source/edge-server/connect.txt

```diff
@@ -201,27 +227,82 @@ You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to it via the host machine using this MongoDB URI:
```

Comment:
Changing it, because I've also been called out for using it, but I do feel "via" is not a foreign word. It's in English language dictionaries and has been accepted into the lexicon. 😛